sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java

```diff
@@ -66,10 +89,43 @@ public MSersDirectorySteps(
 
     When(
         "I select ([^\"]*) disease from Disease combobox",
-        (String disease) -> {
-          webDriverHelpers.selectFromCombobox(DISEASE_COMBOBOX, disease);
+        (String disease) -> webDriverHelpers.selectFromCombobox(DISEASE_COMBOBOX, disease));
+    When(
+        "I select ([^\"]*) from Region combobox",
+        (String disease) -> webDriverHelpers.selectFromCombobox(REGION_COMBOBOX, disease));
+    When(
+        "I select ([^\"]*) from District combobox",
+        (String disease) -> webDriverHelpers.selectFromCombobox(DISTRICT_COMBOBOX, disease));
+    When(
+        "I select ([^\"]*) from Facility combobox",
+        (String disease) -> webDriverHelpers.selectFromCombobox(FACILITY_COMBOBOX, disease));
+    When(
+        "I select ([^\"]*) from Point Of Entry combobox",
+        (String disease) -> webDriverHelpers.selectFromCombobox(POINT_OF_ENTRY_COMBOBOX, disease));
+    When(
+        "I click to Export aggregate report",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CASE_CONTACT_EXPORT);
+          String fileName = "sormas_aggregate_reports_" + LocalDate.now() + "_.csv";
+          FilesHelper.waitForFileToDownload(fileName, 30);
```

Comment:
to all above methods, specify also directory location :D

Reply:
done